product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -78,17 +78,50 @@ Once your identity provider is set up, you can view your connection status, ID,
 You need a verified domain so your users can have a streamlined login experience with their email address. 
 
 1. On the **Domains** tab, enter the domain name and select **Next: Verify Domain**. 
-1. Copy the TXT record and follow the instructions. 
+1. Copy the TXT record and follow the instructions to add it as a TXT record on that domain within your DNS provider's management console.
 1. Select **Done**. 
 
-   Your domain and its status appear on the **Domains** tab, where you can delete or verify it. Domains can take up to 48 hours to be verified. 
+   Your domain and its status appear on the **Domains** tab, where you can delete or verify it. 
+   Domains can take up to 48 hours for the change of the domain record by the DNS provider to propagate before you can verify it. 
+   
+   If your domain doesn't appeared as verified after some time, you can debug whether your domain has the matching verification text field. Query it directly with DNS tools, such as nslookup.
+
+   Select **Verify** next to the domain at `/settings/domains` to check whether this matches the required TXT field. Domains can have many TXT fields. As long as one matches, it works.
```

Comment:
I couldn't follow this step. And should it have its own number? 

Should the code follow the mention of nslookup rather than having this intervening sentence?

Reply:
This is a debug / support step in case the preceding steps have not resulted in a verified domain. 
Plus it illustrates exactly what should have been entered as the TXT field for the domain. I will reword it.

Reply:
OK I have updated.